frontend/src/pages/Player/Toolbar/DevToolsWindowV2/ConsolePage/ConsolePage.tsx

```diff
@@ -82,46 +85,29 @@ export const ConsolePage = ({
 		return data.sessionLogs
 	}, [filter, data?.sessionLogs])
 
-	// Logic for scrolling to current entry.
-	useEffect(() => {
-		if (state !== ReplayerState.Playing || !autoScroll) {
-			return
-		}
-		if (messagesToRender.length) {
-			let cursor = ''
-			let msgDiff: number = Number.MAX_VALUE
-			for (let i = 0; i < messagesToRender.length; i++) {
-				const currentDiff: number =
-					time -
-					(new Date(messagesToRender[i].node.timestamp).getTime() -
-						new Date(messagesToRender[0].node.timestamp).getTime())
-				if (currentDiff < 0) break
-				if (currentDiff < msgDiff) {
-					cursor = messagesToRender[i].cursor
-					msgDiff = currentDiff
-				}
-			}
-			setSelectedCursor(cursor)
-		}
-	}, [state, time, messagesToRender, autoScroll])
-
-	const virtuoso = useRef<VirtuosoHandle>(null)
+	const messageNodes = messagesToRender.map((message) => {
+		return message.node
+	})
 
-	const foundIndex = useMemo(() => {
-		return messagesToRender.findIndex(
-			(logEdge) => logEdge.cursor === selectedCursor,
+	const lastActiveLogIndex = useMemo(() => {
+		return findLastActiveEventIndex(
+			time,
+			sessionMetadata.startTime,
+			messageNodes,
 		)
-	}, [messagesToRender, selectedCursor])
+	}, [messageNodes, sessionMetadata.startTime, time])
```

Comment:
seems like this change to the deps of the useEffect below means that the `Go to` button on logs no longer works

could you see why it's not working? this is a regression from `main` as the button works fine there
<img width="1133" alt="Screenshot 2023-09-12 at 3 57 03 PM" src="https://github.com/highlight/highlight/assets/1351531/d35cb37d-e76e-44d1-a761-219e1bc053f1">